setup.py

```diff
@@ -162,6 +162,7 @@ def run(self):
     cmdclass={"develop": CustomDevelopCommand},
     entry_points={
         "console_scripts": [
+            "xain_fl_coordinator=xain_fl.grpc.cli:main",
```

Comment:
Isn't `coordinator` enough as naming for the entry point?

Reply:
We can name it just `coordinator`. Fine by me. @wilk10 and @janpetschexain whats your opinion?

Reply:
i think `coordinator`is enough